### DISCUSSION TITLE
post/auto-darkmode-on-gnome

### DISCUSSION THREAD

**Discussion by giscus** [GrassBlock1/GrassBlock1#7] [Blog]
# post/auto-darkmode-on-gnome

经过几天的摸索，我最终找到了让 GNOME 桌面环境下随时间自动切换全局可自定义的深色模式的方法，同时也让一部分应用服从系统设定。

https://lab.imgb.space/post/auto-darkmode-on-gnome

**Comment by WeepingDogel** [comment]
来逛逛～
